.github/workflows/lint-test.yml

```diff
@@ -0,0 +1,91 @@
+name: Lint & Test
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  push:
+    branches:
+      - '**'
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Composer dependencies
+        run: composer install --no-interaction --prefer-dist
+      - name: Run PHP Lint
+        run: composer phpcs
+  wporg-validation:
+    name: WP.org Plugin Validation
+    runs-on: ubuntu-latest
+    steps:
+      - uses: pantheon-systems/action-wporg-validator@1.0.0
+        with:
+          type: 'plugin'
+  test-phpunit-74:
+    needs: lint
+    runs-on: ubuntu-latest
+    services:
+      mariadb:
+        image: mariadb:10.5
+    name: PHP 7.4 Unit Tests
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup PHP 7.4
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.4
+          extensions: mysqli, zip, imagick
+      - name: Start MySQL Service
+        run: sudo systemctl start mysql
+      - name: Cache dependencies
+        uses: actions/cache@v3
+        with:
+          path: ~/vendor
+          key: test-phpunit-dependencies-${{ hashFiles('composer.json') }}
+          restore-keys: test-phpunit-dependencies-${{ hashFiles('composer.json') }}
+      - name: Install Composer dependencies
+        run: |
+          composer update && composer install
+      - name: Run PHPUnit
+        run: |
+          bash bin/install-wp-tests.sh wordpress_test root root 127.0.0.1 latest
+          echo "Running PHPUnit on Single Site"
+          composer phpunit
+          rm -rf $WP_TESTS_DIR $WP_CORE_DIR
+          bash bin/install-wp-tests.sh wordpress_test root root 127.0.0.1 nightly true
+          echo "Running PHPUnit on Multisite"
+          WP_MULTISITE=1 composer phpunit
+  test-phpunit-82:
+    needs: lint
+    runs-on: ubuntu-latest
+    services:
+      mariadb:
+        image: mariadb:10.6
+    name: PHP 8.2 Unit Tests
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup PHP 8.2
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 8.2
+          extensions: mysqli, zip, imagick
+      - name: Start MySQL Service
+        run: sudo systemctl start mysql
+      - name: Cache dependencies
+        uses: actions/cache@v3
+        with:
+          path: ~/vendor
+          key: test-phpunit-dependencies-${{ hashFiles('composer.json') }}
+          restore-keys: test-phpunit-dependencies-${{ hashFiles('composer.json') }}
+      - name: Install Composer dependencies
+        run: composer install
+      - name: Run PHPUnit
+        run: |
+          bash bin/install-wp-tests.sh wordpress_test root root 127.0.0.1 latest
+          echo "Running PHPUnit on Single Site"
+          composer phpunit
+          rm -rf $WP_TESTS_DIR $WP_CORE_DIR
+          bash bin/install-wp-tests.sh wordpress_test root root 127.0.0.1 nightly true
```

Comment:
🔢I know this is copied over so optional if you want to adjust, but I believe this should be latest, not nightly?

Reply:
the solr-power script is doing nightly for the second test. I was unsure if that was intentional or not. I'm happy either way.